bq-metrics-extractor/config/config.yaml

```diff
@@ -19,10 +19,23 @@ bqmetrics:
         WHERE event_name = "first_open"
         LIMIT 1000
   - type: summary
+    name: sims_who_have_used_data
+    help: Number of SIMs that has used data last 24 hours
+    resultColumn: count
+    sql: >
+        SELECT count(DISTINCT msisdn) FROM `pantel-2decb.data_consumption.raw_consumption`
+        WHERE timestamp >= TIMESTAMP_SUB(CURRENT_TIMESTAMP(), INTERVAL 1 DAY)
+  - type: summary
+    name: total_data_used
+    help: Total data used last 24 hours
+    resultColumn: count
+    sql: >
+        SELECT sum(bucketBytes) FROM `pantel-2decb.data_consumption.raw_consumption`
```

Comment:
Same as above, we need to name the result column. We are setting the name of the `resultColumn`

Reply:
@la3lma ^^^
Shouldn't we have to name the result column ? `SELECT sum(bucketBytes) as count`